### ISSUE TITLE
"Save", "Save and close", "Save as" won't work with most Gelbooru images

### ISSUE THREAD

**Issue by herpaderphurr** [Bionus/imgbrd-grabber#488]
I'm encountering a problem when browsing Gelbooru, where most images won't work with the save buttons in the image window, like "Save" and "Save and Close". 

Here's an example of an image that works for me:
http://gelbooru.com/index.php?page=post&s=view&id=3079478

But this one doesn't:
http://gelbooru.com/index.php?page=post&s=view&id=3071898

This might be the the same as Falord's issue #487 but I haven't tried other sites.

**Comment by Bionus** [comment]
What do you mean don't work?
The directory is created but the image not saved, as in issue #487?

If it is a new installation (not an update): prior to this happening, did you even once open the options window and click the "Ok" button? (even without changing anything, it seems some default values are not properly set in the last version)

Could you upload your main.log file of after this happens? I can't reproduce this issue.

**Comment by herpaderphurr** [comment]
The directory is created, but the image is not saved. I have opened the options window since the installation. When I try and save something, Grabber indicates that it works - e.g. I'll get the Saved! message, and Save and Close will close the window - but I'll check the directory and it's just not there. 

Here's my main.log. 
http://pastebin.com/jRN04ysU

All I did was open Grabber; search for, open, and save the image that works fine; then search for, open, and (try to) save the image that doesn't work.
